roles/openshift_logging/tasks/upgrade_logging.yaml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Stop the Cluster
+  include: stop_cluster.yaml
+
+- name: Upgrade logging
+  include: install_logging.yaml
+  vars:
+    start_cluster: False
+
+# ensure that ES is running
+- shell: >
+    {{openshift.common.client_binary}} --config={{mktemp.stdout}}/admin.kubeconfig get dc -l component=es -o name -n {{openshift_logging_namespace}}
+  register: es_dc
+  check_mode: no
+
+- name: start elasticsearch
+  include: scale.yaml
+  vars:
+    desired: 1
```

Comment:
Hmm, this should probably look at the number of replicas.. the intent is that we have _at least_ 1 replica